docs/06-concepts/03-serialization.md

```diff
@@ -24,7 +24,19 @@ For most purposes, you will want to use Serverpod's native serialization. Howeve
     }
     ```
 
-3. You must declare your custom serializable objects in the `config/generator.yaml` file in the server project, the path needs to be accessible from both the server package and the client package.
+3. There must be a method called `copyWith()`, which returns a new instance of the object with the specified fields replaced.
```

Comment:
Suggestion: Add a hint that it is implemented as a deep copy in the framework and that we recommend implementing it as a deep copy for custom classes as well.